lib/order.rb

```diff
@@ -0,0 +1,65 @@
+require_relative 'menu'
+require_relative 'text'
+
+class Order
+  include Text
+  attr_reader :orders
+  def initialize
+    @orders = []
+  end
+
+  def greet
+    puts "Thank you for visiting our takeaway website."
+    puts "please take a look at our menu."
+  end
+
+  def present(menu)
+    menu.show
+  end
+
+  def choose_dish
+    puts "What would you like to order? Please enter a menu item number."
+    @menu_num = gets.chomp.to_i
+  end
+  
+  def choose_how_many
+    puts "How many of dishes do you want?"
+    @quantities = gets.chomp.to_i
+  end
+
+  def cart(menu)
+    orders << menu.menu[@menu_num - 1].merge({quantities: @quantities})
+  end
+
+  def check_orders
+    width = 10
+    puts "Dish".ljust(width) + "Quantities".center(width) +
+      "Price".center(width) + "Total".rjust(width)
+    orders.each do |order|
+      puts "#{order[:dish].ljust(width)}" +
+      "#{order[:quantities].to_s.center(width)}" +
```

Comment:
Style/LineEndConcatenation: Use `\` instead of `+` or `<<` to concatenate those strings.<br>Style/MultilineOperationIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.